src/Lumi/Components/Input.purs

```diff
@@ -372,14 +377,20 @@ styles = jss
                       , lineHeight: "1"
                       }
                   }
+              , "&::before":
+                  { content: "'•'"
```

Comment:
Unfortunately, if this were `content: "' '"`, aligning by baseline wouldn't work. It seems that browsers don't consider whitespace as characters when aligning stuff :/